src/modules/MouseUtils/MouseHighlighter/MouseHighlighter.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 #include "pch.h"
 
-constexpr int MOUSE_HIGHLIGHTER_DEFAULT_OPACITY = 65;
-const winrt::Windows::UI::Color MOUSE_HIGHLIGHTER_DEFAULT_LEFT_BUTTON_COLOR = winrt::Windows::UI::ColorHelper::FromArgb(MOUSE_HIGHLIGHTER_DEFAULT_OPACITY, 255, 255, 0);
-const winrt::Windows::UI::Color MOUSE_HIGHLIGHTER_DEFAULT_RIGHT_BUTTON_COLOR = winrt::Windows::UI::ColorHelper::FromArgb(MOUSE_HIGHLIGHTER_DEFAULT_OPACITY, 0, 0, 255);
+const winrt::Windows::UI::Color MOUSE_HIGHLIGHTER_DEFAULT_LEFT_BUTTON_COLOR = winrt::Windows::UI::ColorHelper::FromArgb(65, 255, 255, 0);
+const winrt::Windows::UI::Color MOUSE_HIGHLIGHTER_DEFAULT_RIGHT_BUTTON_COLOR = winrt::Windows::UI::ColorHelper::FromArgb(65, 0, 0, 255);
+const winrt::Windows::UI::Color MOUSE_HIGHLIGHTER_DEFAULT_ALWAYS_COLOR = winrt::Windows::UI::ColorHelper::FromArgb(0, 255, 0, 0);
```

Comment:
I don't think these values are 65%, which was the default before... They seem to be closer to 25%. (it's relative to 255)

Reply:
But now that I've checked. that seems to be the new default since changing to percent earlier. I guess this is what we've actually been using as default (25%), even if unintended.

Reply:
Aha, I felt the color was more transparent than when I first tried MouseHighlighter when it was introduced.

I think this is a good opportunity to fix that too.

I changed opacity values to 65%.
Because 65 * 2.55 == 165.75, the new value is 166 (10) == 0xa6 (16).